rpc/nodeagent/compute/virtualmachine/moc_nodeagent_virtualmachine.proto

```diff
@@ -31,10 +31,17 @@ message Disk {
 	string diskname = 1;
 }
 
+message SharedFolder {
```

Comment:
Is there a md file for this feature?

Reply:
yes, the design doc in review [https://github.com/jagadishmurugan/wssdagent/pull/2](https://github.com/jagadishmurugan/wssdagent/pull/2)